dff/pipeline/pipeline/pipeline.py

```diff
@@ -217,7 +229,7 @@ def from_script(
         parallelize_processing: bool = False,
         handlers: Optional[Dict[ActorStage, List[Callable]]] = None,
         context_storage: Optional[Union[DBContextStorage, Dict]] = None,
-        messenger_interface: Optional[MessengerInterface] = None,
+        messenger_interfaces: Optional[Union[MessengerInterface, Iterable[MessengerInterface], Dict[str, MessengerInterface]]] = None,
```

Comment:
I don't think we should allow passing `MessengerInterface` instead of iterables.